scripts/generate_exp.py

```diff
@@ -0,0 +1,65 @@
+"""
+ Copyright (C) 2023 Intel Corporation
+
+ Part of the Unified-Runtime Project, under the Apache License v2.0 with LLVM Exceptions.
+ See LICENSE.TXT
+ SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+"""
+import sys
+from util import makoWrite
+import re
+
+usage = """
+OVERVIEW: A tool to aid in adding a new experimental feature to the Unified Runtime Specification.
+
+USAGE: python generate_exp.py [options] <name>
+
+    <name> - must be lowercase and kebab case i.e. command-buffer
+
+OPTIONS:
+    --help, -h              - Display usage and valid options available
+"""
+
+if __name__ == "__main__":
```

Comment:
I'd recommend using the following pattern in general, it makes using the `main()` function outside of this script possible and is clearer to read.

```python
def main():
   # implementation


if __name__ == '__main__':
    try:
        main()
    except KeyboardInterrupt:
        exit(130)
```

Reply:
👍